### PR TITLE
added stored procedures to be included in the dump

### PR DESCRIPTION
**Pull request by ksairi**
#82

### CODE REVIEW

**Comment by ksairi** [comment]
@bradzacher thank you for this library, we have been using it in our company for some years already! We are interested in backuping stored procedures now and I have been working on that feature to have a decent PR to provide.
I have based my coding on `triggers` part to build similar architecture, but I noticed that the tests fail before putting my code so it has been pretty difficult for me to reach to a 100% tests passed. Would you help on this PR? we are looking forward to start backuping our SPs soon!
cheers!

**Comment by bradzacher** [comment]
> but I noticed that the tests fail before putting my code 

Unless something has changed, that isn't correct. The last master build is green, so the tests are fine.

https://travis-ci.org/bradzacher/mysqldump/builds/563146259?utm_source=github_status&utm_medium=notification

---

You can view the output from your CI run in the travis web UI - just click the details link at the bottom of the PR.
image

https://travis-ci.org/bradzacher/mysqldump/jobs/590634977?utm_medium=notification&utm_source=github_status

Looks like there's a type error right now.

File: src/getProcedureDump.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    character_set_client: string;
+    coallation_connection: string;
+    'Database Collation': string;
+}
+interface ShowCreateProcedure {
+    Procedure: string;
+    sql_mode: string;
+    'Create Procedure': string;
+    character_set_client: string;
+    coallation_connection: string;
```
Comment (yanisalfian) [inline src/getProcedureDump.ts:17]: Wrong definition. It should be ```collation_connection```
```suggestion
    collation_connection: string;
```

**Review by yanisalfian — COMMENTED** [review]
Wrong type definition in ```getProcedureDump.ts```

**Pull request closed without merging**